docs/Using-Fleet/ChromeOS.md

```diff
@@ -0,0 +1,23 @@
+# ChromeOS
+
+## Adding ChromeOS hosts to Fleet
+Fleet provides a Chrome extension which you can install via Google Admin.
+
+To learn how to add ChromeOS hosts to Fleet, visit [here](https://fleetdm.com/docs/using-fleet/adding-hosts#add-chromebooks-with-the-fleetd-chrome-extension).
+
+## Available tables
+To see the available tables for ChromeOS, visit [here](https://fleetdm.com/tables).
```

Comment:
Looks good, suggestion for future iteration: maybe @eashaw can build the tables to keep the platform type in a url param so we can directly link to chromeos tables only (e.g., `https://fleetdm.com/tables/account_policy_data&platform=chromeos`) 

@zhumo I'll let you file and prioritize if you agree

What that would look like:
<img width="1442" alt="Screenshot 2023-06-09 at 9 24 15 AM" src="https://github.com/fleetdm/fleet/assets/71795832/ab3adf90-a64e-401d-ab1f-b24a6cd76202">

Vs. what we currently see clicking this link
<img width="1435" alt="Screenshot 2023-06-09 at 9 24 52 AM" src="https://github.com/fleetdm/fleet/assets/71795832/37c085ac-1fdd-4439-875f-71d892371693">

Reply:
@zhumo actually, this link works better since it has the platform filtered to chrome: https://fleetdm.com/tables/chrome_extensions?platformFilter=chrome

Can we swap https://fleetdm.com/tables?

Reply:
oh thanks! I was looking for this. When I click the platform filter, the URL params don't change, so I assumed that doesn't exist! will change.

Reply:
```suggestion
To see the available tables for ChromeOS, visit [here](https://fleetdm.com/tables/chrome_extensions?platformFilter=chrome).
```